src/Orm/Zed/AppKernel/Persistence/SpyAppConfig.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Orm\Zed\AppKernel\Persistence;
+
+use Orm\Zed\AppKernel\Persistence\Base\SpyAppConfig as BaseSpyAppConfig;
+
+/**
+ * Skeleton subclass for representing a row from the 'spy_app_config' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyAppConfig extends BaseSpyAppConfig
```

Comment:
if ORM classes (from other packages) are needed only for CI, let's generate them in CI

Reply:
Committed in suite as well.

Reply:
it's not a suite (project code), it's a library package that cannot be used as a standalone project.